packages/index.css

```diff
@@ -7,6 +7,7 @@
 @import 'components/card.css';
 @import 'components/list.css';
 @import 'components/navigation.css';
```

Comment:
Suggestion (do later)
Make Navigation bar responsive

Else everything looks fine!!
Good to merge